### DISCUSSION TITLE
Impossible to modify padding/margin/height/etc.

### DISCUSSION THREAD

**Discussion by florentgl** [tailwindlabs/tailwindcss#12679] [Help]
Hi guys !

I'm currently learning Tailwind, and I began a training project like a fake portfolio page. 
Everything was alright at the beginning of the coding, Tailwind was working well, I was able to see all my changes on my live server.

But, at a moment, all my margins/paddings/etc. were not working anymore.
All the other tailwind classes are working well, but not these ones, I don't understand why.

I checked few times my code, to see if it was something in the structure, maybe a mistake that I've done somewhere, but can't find anything.

For example, I can't see these changes on my live server : `<div class="container mt-7 mx-10">,` and neither these :          ` <svg class="ml-4" xmlns="http://www.w3.org/2000/svg" width="26" height="23">
            <path fill="#FFF"
              d="M13.084.23a.751.751 0 0 0-.736.75v14.267a.75.75 0 1 0 1.5 0V.98a.75.75 0 0 0-.763-.75ZM24.44 4.504a.752.752 0 0 0-.284.064l-6.44 2.875a.752.752 0 0 0 0 1.37l6.44 2.884a.75.75 0 0 0 .612-1.369L19.861 8.13l4.907-2.191a.753.753 0 0 0 .38-.99.752.752 0 0 0-.708-.444ZM1.371 9.663a.752.752 0 0 0-.74.938C2.41 17.447 8.603 22.23 15.685 22.23a.75.75 0 1 0 0-1.501A14.053 14.053 0 0 1 2.083 10.225a.75.75 0 0 0-.712-.561v-.001Z" />
          </svg>`

Anybody knows what happen here ?

https://github.com/florentgl/learn-tailwind

Thanks guys ;)

**Comment by wongjn** [comment]
You've introduced an empty object at `theme.spacing` in your Tailwind configuration. This eliminates all default `spacing` values that `padding`, `margin`, `height` et al. use. If you don't need to add any spacing values, consider removing `theme.spacing` from your configuration. If you would like to add spacing values in the future while keeping the default ones, define your spacing values in `theme.extend.spacing` instead of `theme.spacing`. Consider [reading more about theme configuration in the documentation](https://tailwindcss.com/docs/theme#customizing-the-default-theme).

**Reply by florentgl** [reply]
Thanks Justin, it was totally that !
Still a lot of things to learn ;)
Thanks for your help !
